m4_sessions/automation/session_one/eight_mike_justin_jesse_ethan_dan.md

```diff
@@ -1,3 +1,53 @@
 ## Group Eight's Session Plan
 
 M4 Student Leaders: Mike, Justin, Jesse, Ethan, Dan
+
+# Turing Career Development: Automation
+
+## Session Outcomes
+
+## Mission Alignment		 
+  		  
+*Our mission is to unlock human potential by training a diverse, inclusive student body to succeed in high-fulfillment technical careers.
+  		  
+As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
+
+## Pre-Session
+
+
+### Survey Questions
```

Comment:
@case-eee I will be taking point on creating and distributing the survey. We will send it out with the materials 24 hrs in advance but will also give time at the beginning to get answers in if you have not while we read out others.